dashboard/src/components/jobs/JobResults.vue

```diff
@@ -46,6 +46,9 @@ Component for handling display/download of job results.
       <template v-else-if="jobStatus == 'running'">
         The calculation is running and will be ready soon.
       </template>
+      <template v-else-if="jobStatus == 'error'">
+        An error occured while processing the calculation.
+      </template>
```

Comment:
also add something for complete. I've noticed the "Calculation has not been submitted" show up before the result loads